label_studio/organizations/models.py

```diff
@@ -94,7 +121,7 @@ def has_project_member(self, user):
         return self.projects.filter(members__user=user).exists()
 
     def has_permission(self, user):
-        if self in user.organizations.all():
+        if self in user.organizations.filter(organizationmember__deleted_at__isnull=True):
```

Comment:
I see that `if self in user.organizations...` is how this was being handled before, but I think this is suboptimal since it'll pull in all organizations the user is a member of when when we could just have the db handle the querying (I don't _think_ django magics a good query out this)

Something like `return OrganizationMember.objects.filter(user=user, organization=self, deleted_at__isnull=True).exists()` should be better.